tests/test_integrations/test_nessus.py

```
@@ -76,3 +78,67 @@ def test_integration_active_invalid_url(self, sample_integration_dict):
+
+        class MockClientInvalidJSON:
+            def list_scans(self):
+                return MockResponseInvalidJSON()
+
+        res = executor({"clients": {"nessus": MockClientInvalidJSON()}})
+        assert res == {"error": "Invalid JSON payload in response"}
+
+        # 4. Invalid response schema (not dict or scans not list)
+        class MockResponseInvalidSchema:
+            status_code = 200
+            def json(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `json`

(ANN202)